### ISSUE TITLE
Order Post is Successful but Status is Inactive

### ISSUE THREAD

**Issue by stamzid** [Voyz/ibeam#139]
**Describe the bug**
I have a docker container running on an AWS ubuntu machine. I am able to subscribe to websocket and interact with API. When I place an order via TWS Workstation I can see the order coming through the websocket client in real time which is great. 

The problem happens when I try to place an order through iserver API. I can see the order payload in the websocket and it is always "Inactive". Have you encountered this before? I have double checked the order body many times and it has required fields. For example, 

{
"orders": [{
"acctId": "<acctId>",
"conid": 341505205,
"secType": "341505205:STK",
"orderType": "LMT",
"listingExchange": "SMART",
"side": "BUY",
"ticker": "IDEX",
"tif": "GTC",
"quantity": 5,
"useAdaptive": false,
"price": 0.04,
"outsideRTH": false
}]
}

I can see the post is successful but order status is inactive

**To Reproduce**
Have ibeam docker running and post through place order iserver api endpoint.

**Expected behavior**
```'status': 'PreSubmitted', 'order_ccp_status': 'Pending Submit',```

**Environment**
IBeam version: latest
Docker image or standalone:
Python version (standalone users only):
OS: Ubuntu

**Additional context**

**Suggest a Fix**
If you can't fix the bug yourself, perhaps you can point to what might be causing the problem (line of code or commit).

**Comment by stamzid** [comment]
I have two areas of suspicion:

1. I'm doing an unverified https request since it's within my own vpc on AWS
2. Maybe I'm missing a flag that transmits the order from Client Portal Web API to interactive broker. With TWS desktop workstation python client I'm able to post orders using the ibsync library. There's an intermediate function call that (qualify contract), I don't know if that's needed.

**Comment by Voyz** [comment]
Hey @stamzid thanks for describing your issue in so much detail 👍

I haven't encountered that issue myself, but here are some of my thoughts:

1. I don't think it would have anything to do with doing an unverified HTTPS. It's just a HTTP-related quirk, shouldn't affect how their API receives and processes your request.
1. Possibly you're missing a flag, yeah. Did you try reaching IBKR about this? I'd imagine their devs team should have some idea as to why does this happen
1. Do you get any response when submitting that order? Does it carry anything useful there?

**Comment by stamzid** [comment]
I reached out to IBKR support and they were able to confirm that I am missing a step. So I missed the confirmation part with reply ID to the `/iserver/reply` endpoint. I actually saw it while debugging but assumed it is needed for only orders that varies a lot from the market price. So now for posting through ibeam we need to do 2 REST calls, one for posting the order and then another post for confirmation.
